src/bauhaus/bauhaus.c

```diff
@@ -714,7 +715,7 @@ float dt_bauhaus_slider_get_hard_min(GtkWidget* widget)
 {
   dt_bauhaus_widget_t *w = DT_BAUHAUS_WIDGET(widget);
   dt_bauhaus_slider_data_t *d = &w->data.slider;
-  return d->hard_min;
+  return (d->callback == NULL ? d->hard_min : d->callback(d->hard_min, DT_BAUHAUS_GET));
```

Comment:
I would just add a linear default callback.
Would save one the trouble of thinking whether there is a callback.